frameworks/JavaScript/ultimate-express/routes.js

```diff
@@ -0,0 +1,253 @@
+        });
+    
+        res.real_end = res.end;
+    
+        res.end = (data) => {
+            
+            let toRet;
+            const json = JSON.parse(data.toString('utf8'));
+    
+            if(json.data.helloWorld) {
+                toRet = json.data.helloWorld;
+            } else {
+                toRet = { helloWorld: null };
+            }
+            
+            setResponseHeaders(res, toRet.length);
+    
+            res.real_end(toRet);
+        }
+    
+        await graphql(req, res);
+    });
+    
+    app.get('/db', async (req, res) => {
+        
+        const graphql = await graphqlHTTP((req, res, graphQLParams) => {
+        
+            graphQLParams.query = "{ singleDatabaseQuery { id, randomNumber }}";
+            graphQLParams.variables = {};
+            graphQLParams.operationName = null;
+        
+            return graphqlOpts();
+        });
+    
+        formatResData(res, "singleDatabaseQuery");
+    
+        await graphql(req, res);
+    });
+    
+    app.get('/queries', async (req, res) => {
+        
+        const graphql = await graphqlHTTP((req, res, graphQLParams) => {
+            
+            let totalNumOfQueries = ensureQueryIsAnInt(req.query.queries);
+    
+            graphQLParams.query = `{ multipleDatabaseQueries(total: ${totalNumOfQueries}) { id, randomNumber }}`
+            graphQLParams.variables = {};
+            graphQLParams.operationName = null;
+    
+            return graphqlOpts();
+        });
+    
+        formatResData(res, "multipleDatabaseQueries");
+    
+        await graphql(req, res);
+    });
+    
+    app.get('/fortunes', async (req, res) => {
+    
+        const graphql = await graphqlHTTP((req, res, graphQLParams) => {
+    
+            graphQLParams.query = "{ getAllFortunes { id, message } }"
+            graphQLParams.operationName = null;
+            graphQLParams.variables = {};
+    
+            return graphqlOpts();
+        });
+    
+        retrieveAndFormatAllFortunes(res);
+        
+        await graphql(req, res);
+    });
+    
+    app.get('/updates', async (req, res) => {
+    
+        const totalNumOfQueries = ensureQueryIsAnInt(req.query.queries);
+    
+        const graphql = await graphqlHTTP((req, res, graphQLParams) => {
+    
+            graphQLParams.query = `{ getRandomAndUpdate(total: ${totalNumOfQueries}) { id, randomNumber } }`
+            graphQLParams.operationName = null;
+            graphQLParams.variables = {};
+    
+            return graphqlOpts();
+        });
+    
+        formatResData(res, 'getRandomAndUpdate');
+    
+        await graphql(req, res);
+    });
+    
+    app.get('/plaintext', (req, res) => {
+    
+        const responseTxt = "Hello, World!";
+    
+        res.setHeader('Content-Length', responseTxt.length);
+        res.setHeader('Server', 'UltimateExpress-GraphQL-MySQL');
+        res.contentType('text/plain');
+        res.status(200);
+    
+        res.send(responseTxt);
```

Comment:
If Content-Length is not specifically needed here and Chunked Transfer encoding is fine, then 
```js
 app.get('/plaintext', (req, res) => {
        res.setHeader("Server", "UltimateExpress-GraphQL-MySQL");
        res.setHeader("Content-Type", "text/plain");

        res.send("Hello, World!");
});
```
would compile it into declarative